compose/django.md

```diff
@@ -105,7 +105,7 @@ In this step, you create a Django starter project by building the image from the
 the [docker-compose run](reference/run.md) command as follows.
 
    ```console
-   $ sudo docker-compose run web django-admin startproject composeexample .
+   $ sudo docker-compose run web django-admin startproject compose example .
```

Comment:
@charleswhchan The django topic has several instances where the project name is set as `composeexample`. Changing just one instance would be incorrect. Could you take a look?